docs/DeveloperGuide.md

```diff
@@ -564,33 +564,40 @@ end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline
+| `* * *`   | pre-event marketer who handles many marketing events  | edit marketing events                                                                                            | keep the system up to date with changes that happen to the marketing plans that are present in the system                                                        |
+| `* * *`   | pre-event marketer who handles many marketing events  | delete marketing events                                                                                          | remove marketing events that I no longer need                                                                                                                    |
+| `* * `    | pre-event marketer who handles many marketing events  | sort marketing events by date                                                                                    | so that I can take a look at when last few marketing events are scheduled and start planning ahead for more if those plans are to be executed soon               |
+| `* * `    | pre-event marketer who handles many marketing events  | sort marketing events by name                                                                                    | so that I can locate events more easily                                                                                                                          |
+| `* *`     | post-event marketer with many marketing events        | generate statistics of marketing events based on the gender and age demographics of my event attendees           | evaluate the success of my marketing campaigns                                                                                                                   |
+| `* *`     | post-event marketer with many marketing events        | generate a mailing list of people who attended the event                                                         | reach out to the people for feedback about the event                                                                                                             |
 
-*{More to be added}*
 
 ### Use cases
 
 (For all use cases below, **UC** is the `Use Case`, the **System** is `Duke The Market` and the **Actor** is the `user`, unless specified otherwise)
 
-**UC01: Delete a customer**
+**UC01: Delete an event**
 
 **MSS**
 
-1. User requests to list customers
-2. Duke The Market shows a list of customers
-3. User requests to delete a specific customer in the list
-4. Duke The Market deletes the customer
+1. User requests to list events
+2. Duke The Market shows a list of events
+3. User requests to delete a specific event in the list
+4. Duke The Market deletes the event
 
    Use case ends.
 
```

Comment:
for below for UC1 2a, should put that Duke The Market will show an error message as well

Reply:
Because when the list if empty calling listEvents just returns an empty list and output message is "Listed all events sorted by no field" so might be incorrect to label it as an error